test/e2e/prune_test.go

```diff
@@ -221,6 +223,14 @@ var _ = Describe("Podman prune", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(ExitCleanly())
 
+		// FIXME: there's a race here: the container process may still be
+		// initializing, and not yet have set up signal handlers when we
+		// send the stop. Minimize that by, sigh, sleeping. (Got a better idea?)
+		// Justification: 1/2 second extra in all CI runs is better than
+		// occasional flakes.
+		// See #20196
+		time.Sleep(500 * time.Millisecond)
+
 		session = podmanTest.Podman([]string{"pod", "stop", podid1})
```

Comment:
we do not care how it is stopped for this test so just use `-t0` here

Reply:
Fair enough. So done.